Create readme with basic instructions (#18)
OWASP/www-project-mobile-security: README.md

> # Mobile Security project web page repo

Reviewer: ```suggestion
# Mobile Security Project Web Page Repo
```

This is the repo for https://owasp.org/www-project-mobile-security/. It is maintained by multiple people at the moment, but no owner has been assigned.

## Changes

> Want to change the page? Follow the same steps as with the MSTG repo: Fork this project, apply changes to your own Fork and then create a PR. The process is similar to contributing to the [MSTG](https://github.com/OWASP/owasp-mstg).

Reviewer: ```suggestion
Want to change the page? Follow the same steps as with the MSTG repo: fork this project, apply changes to your own fork and then create a PR. The process is similar to contributing to the [MSTG](https://github.com/OWASP/owasp-mstg).
```

> ## Report an issue

Reviewer: ```suggestion
## Report an Issue
```

> Want to report an issue with the content or how it is displayed? File an [issue](https://github.com/OWASP/www-project-mobile-security/issues). ** Please note that this project is only about having the proper links and guidance towards the other projects. It is not responsible for the content of the other project page. For this, you have to look for the leader of that given project. **

Reviewer: ```suggestion
Want to report an issue with the content or how it is displayed? File an [issue](https://github.com/OWASP/www-project-mobile-security/issues). **Please note that this project is only about having the proper links and guidance towards other mobile related projects. It is not responsible for the content of the other project pages. For this, you have to look for the leader of that specific project.**
```

## Communication

> Currently most of the communication can best be done via OWASP Slack. See [the MSTG](https://github.com/OWASP/owasp-mstg) on how ot get there. Please understand that OWASP has a global code of conduct to adhere to, which is similar as the one used by the [MSTG](https://github.com/OWASP/owasp-mstg/blob/master/CODE_OF_CONDUCT.md).

Reviewer: ```suggestion
Currently most of the communication can best be done via OWASP Slack. See [the README in the MSTG repo](https://github.com/OWASP/owasp-mstg) on how to get there. Please understand that OWASP has a global code of conduct to adhere to, which is similar to the one used by the [MSTG](https://github.com/OWASP/owasp-mstg/blob/master/CODE_OF_CONDUCT.md) and must be followed in any circumstance.
```
